app/src/main/java/com/example/tasky/auth/presentation/register/RegisterScreen.kt

```diff
@@ -92,146 +106,269 @@ private fun RegisterScreen(
 ) {
     val passwordState = rememberTextFieldState(initialText = state.password)
 
+    val windowSizeClass = currentWindowAdaptiveInfo().windowSizeClass
+    val deviceConfiguration = DeviceConfiguration.fromWindowSizeClass(windowSizeClass)
+
     LaunchedEffect(passwordState.text) {
         val newPassword = passwordState.text.toString()
         if (newPassword != state.password) {
             onAction(RegisterAction.OnPasswordValueChanged(newPassword))
         }
     }
 
-    TaskyScaffold(
```

Comment:
question: Any reason to  replace TaskyScaffold with a stock Scaffold here?  🤔 If TaskyScaffold provides shared theming/insets/semantics, consider enhancing it to cover adaptive needs.